api/src/main/kotlin/com/gmtkgamejam/plugins/AdminRoutes.kt

```diff
@@ -0,0 +1,51 @@
+package com.gmtkgamejam.plugins;
+
+import com.gmtkgamejam.models.PostItem
+import com.gmtkgamejam.models.admin.DeleteTeamDto
+import com.gmtkgamejam.models.admin.ReportedUsersClearDto
+import com.gmtkgamejam.services.PostService
+import io.ktor.application.*
+import io.ktor.http.*
+import io.ktor.request.*
+import io.ktor.response.*
+import io.ktor.routing.*
+import org.litote.kmongo.gt
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+fun Application.configureAdminRouting() {
+
+    val service = PostService()
+
+    routing {
+        route("/admin") {
+            route("/reports") {
+                get {
+                    call.respond(service.getPosts(PostItem::reportCount gt 0))
+                }
+                post("/clear") {
```

Comment:
Routes like this should probably be guarded by authentication (eventually that is)
https://github.com/GameMakersToolkit/team-finder/blob/45b484092e4bf0375d3f46966dc04a0d650aeb2f/api/src/main/kotlin/com/gmtkgamejam/plugins/Routing.kt#L29